source/features/link-to-changelog-file.tsx

```diff
@@ -11,11 +11,13 @@ import {buildRepoURL, getRepo} from '../github-helpers';
 
 const getCacheKey = (): string => `changelog:${getRepo()!.nameWithOwner}`;
 
-function parseFromDom(): string | false {
-	return select('.js-navigation-item [title^="changelog." i]')?.textContent ?? false;
+async function parseFromDom(): Promise<false> {
+	await cache.delete(getCacheKey());
```

Comment:
I don’t think this should be necessary. .set already overwrites it. 

If you copied this exact piece of code from other features, they should probably be updated too, but I think those used cache.delete because they called cache.function afterwards, not cache.set

Reply:
> but I think those used cache.delete because they called cache.function afterwards, not cache.set

correct